pegasus/sites.v3/hourofcode.com/public/index.haml

```diff
@@ -4,7 +4,7 @@ title: "<%= hoc_s(:front_title).gsub(/%{campaign_date}/, campaign_date('full-yea
 social:
   "og:title": "<%= hoc_s(:meta_tag_og_title) %>"
   "og:description": "<%= hoc_s(:meta_tag_og_description) %>"
-  "og:image": "https://<%=request.host%>/images/hourofcode-2015-video-thumbnail.png"
+  "og:image": "https://hourofcode.com/images/hourofcode-2015-video-thumbnail.png"
```

Comment:
Yeah I just removed the request.host to be consistent. I think that was probably copied from code.org/csedweek.org when it was shared. But I don't think needed here?